docs/deploy-full-rp-service-in-dev.md

```diff
@@ -398,3 +398,38 @@
     ```
 
     > __NOTE:__ The `az aro` CLI extension must be registered in order to run `az aro` commands against a local or tunneled RP. The usual hack script used to create clusters does not work due to keyvault mirroring requirements. The name of the cluster depends on the DNS zone that was created in an earlier step.
+
+## Recover VPN access
+
+Since setting up your own VPN in an earlier step will overwrite your local secrets, you will lose access to the vpn / vnet gateway that you provisioned in an earlier step if you run `make secrets`. If you don't have a secrets/* backup, you can recover your access using the following steps. Please note that this action will _**OVER WRITE**_ the `secrets/vpn-$LOCATION.ovpn` on your local machine. **DO NOT** run `make secrets-update` after doing this, as you will overwrite the shared secrets for all users.
+
+1. Source all environment variables from earlier, and run the VPN configuration step again:
+
+    ```bash
+    . ./env
+    . ./env-int
+
+    source ./hack/devtools/deploy-shared-env.sh
```

Comment:
Nit pick, I would prefer either `.` or `source` is used, rather than a mix of the two.

Reply:
done!